sdk/cognitiveservices/cognitiveservices-luis-authoring/README.md

```diff
@@ -2,11 +2,6 @@
 
 This package contains an isomorphic SDK for LUISAuthoringClient.
 
-Package version | LUIS Authoring API version
---------------- | --------------------------
-3.0.0           |  /luis/api/v2.0
-4.0.0-preview.3 |  /luis/authoring/v3.0-preview
```

Comment:
This part was manually added by the LUIS team to clarify which latest version of the package targets which version of the LUIS APIs

Please retain this and update the versions appropriately